### PR TITLE
Rhood/2528 changes to all in one schema

### PR DESCRIPTION
**Pull request by rhood23699**
This PR contains all modifications to the All-In-One-Health Schema to:

- Set the patient_county cardinality to ZERO_OR_ONE to prevent an entire file from failing if there is an invalid zip code sent, or Zip CSV file is missing zip codes.
- OBX.24.3 thru OBX.24.5 - Set Performing Lab city, state and zip to the values in Columns D, E and F from the CSV file.

**If you are suggesting a fix for a currently exploitable issue, please disclose the issue to the prime-reportstream team directly outside of GitHub instead of filing a PR, so we may immediately patch the affected systems before a disclosure. See [SECURITY.md/Reporting a Vulnerability](https://github.com/CDCgov/prime-reportstream/blob/master/SECURITY.md#reporting-a-vulnerability) for more information.**

Test Steps:
1. Re-run the existing All-In-One-Health test file the local RS environment and validate that the OBX.24.3 thru OBX.24.5
2. Alter a patient zip code in the test file.  Process it and validate that the file did not fail.

## Changes
- Set the patient_county cardinality to ZERO_OR_ONE to prevent an entire file from failing if there is an invalid zip code sent, or Zip CSV file is missing zip codes.
- OBX.24.3 thru OBX.24.5 - Set Performing Lab city, state and zip to the values in Columns D, E and F from the CSV file.

### Testing
- [x] Tested locally?
- [x] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] Downloaded a file from `http://localhost:7071/api/download`?
- [ ] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Fixes
*List GitHub issues this PR fixes*
- #2528  

## To Be Done
*Create GitHub issues to track the work remaining, if any*

### CODE REVIEW

File: prime-router/metadata/schemas/non-standard/all-in-one-health-ca-covid-19.schema [hunk 1]
```diff
@@ -129,14 +130,19 @@ elements:
       documentation: Translate inbound text to outbound SNOMED Codes
       csvFields: [{ name: Specimen Type, format: $display}]
 
+
+# Set fields needed for the LIVDlookup() Mapper to set the LOINC code and OBX.17 - Device Info
+    - name: equipment_model_name
+      csvFields: [{name: Device Identifier}]
+#      mapperOverridesValue: true
```
Comment (carlosfelix2) [inline prime-router/metadata/schemas/non-standard/all-in-one-health-ca-covid-19.schema:137]: Remove commented out lines.

File: prime-router/metadata/schemas/non-standard/all-in-one-health-ca-covid-19.schema [hunk 2]
```diff
@@ -129,14 +130,19 @@ elements:
...
+
     - name: test_performed_code
       type: TEXT
       csvFields: [{name: Test Code}]
 
-    - name: equipment_model_name
-      type: TEXT
-      default: 99999999
-      csvFields: [{name: Device Identifier}]
+#    - name: test_kit_name_id
```
Comment (carlosfelix2) [inline prime-router/metadata/schemas/non-standard/all-in-one-health-ca-covid-19.schema:143]: Remove commented out lines

File: prime-router/metadata/schemas/non-standard/all-in-one-health-ca-covid-19.schema [hunk 3]
```diff
@@ -129,14 +130,19 @@ elements:
...
+#      type: TEXT
```
Comment (carlosfelix2) [inline prime-router/metadata/schemas/non-standard/all-in-one-health-ca-covid-19.schema:144]: Remove commented out lines

Comment (rhood23699) [reply prime-router/metadata/schemas/non-standard/all-in-one-health-ca-covid-19.schema:144]: Leaving comments for now to reference again if needed during testing in Staging.

**Pull request merged**